### PR TITLE
Adds new theme [JOHLC/transparentblue]

### PR DESCRIPTION
**Pull request by JOHLC**
Add Transparent Blue to HACS theme repo

### CODE REVIEW

**Comment by JOHLC** [comment]
Looks like there was an issue with validation platform last night, this is working now and I hope it can be merged

**Comment by JOHLC** [comment]
Latest release v0.9

File: theme [hunk 1]
```diff
@@ -36,6 +36,7 @@
   "home-assistant-community-themes/teal",
   "home-assistant-community-themes/vaporwave-pink",
   "houtknots/UglyChristmas-Theme",
+  "JOHLC/transparentblue",
```
Comment (JOHLC) [inline theme:39]: Adding transparentblue repo to theme file

Comment (KTibow) [reply theme:39]: Why are you making a review?

**Comment by JOHLC** [comment]
Github suggested it, didn't realize I shouldn't

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Pull request merged**